client/src/components/QA/QAAddAnswerModal.jsx

```diff
@@ -21,28 +13,35 @@ const QAAddAnswerModal = ({ question, productName }) => {
 
   return (
     <div data-testid="qa-div-card-questions">
-      <button onClick={() => setModalIsOpenAdd(true)} className="qa-add-answer" href=""><b>[Add Answer]</b></button>
-      <Modal isOpen={modalIsOpenAdd} onRequestClose={() => setModalIsOpenAdd(false)}>
-        <h2>Submit Your Answer</h2>
-        <h3>{productName.name}: {question.question_body} </h3>
-        <form className="add-answer-form">
-          <label className="add-form-answer">Answer* </label>
-            <textarea rows="10" maxLength="1000" required onChange={() => setAnswer(event.target.value)}/><p></p>
-          <label className="add-form-username">Nickname*</label>
-            <input maxLength="60" placeholder="Example: jack543!" required onChange={() => setNickname(event.target.value)}/><p className="qa-form-small">For privacy reasons, do not use your full name or email address</p>
-          <label className="add-form-email">Email*</label>
-            <input maxLength="60" placeholder="Example: jack@email.com" required onChange={() => setEmail(event.target.value)}/><p className="qa-form-small">For authentication reasons, you will not be emailed</p>
-          <button className="qa-add-form-submit" onClick={() => {
-            event.preventDefault();
-            setModalIsOpenAdd(false);
-            setModalInfo({
-              answer: answer,
-              nickname: nickname,
-              email: email
-            });
-          }}>Submit</button>
-        </form>
-      </Modal>
+      <button id="qa-modal-button" className="qa-add" onClick={() => {
+        document.getElementById(index).style.display = "block";
+      }}><b>[Add Answer]</b></button>
+      <div id={index} className="qa-modal">
+        <div className="qa-modal-content">
+          <div className="qa-modal-close" onClick={() => {
+            document.getElementById(index).style.display = "none";
+          }}>&times;</div>
+          <h2>Submit Your Answer</h2>
+          <h3>{productName.name}: {question.question_body} </h3>
+          <form className="add-answer-form">
+            <label className="add-form-answer">Answer* </label>
+            <textarea rows="10" maxLength="1000" required onChange={() => setAnswer(event.target.value)} /><p></p>
```

Comment:
For spacing, instead of using an empty paragraph, I would just style the textarea element in the style CSS. Or maybe wrap each label/input pair in their own divs, which would be easier to select, and considering they should be paired together like that.